plugins/include/message_const.inc

```diff
@@ -672,6 +720,12 @@ enum
  */
 #define TE_LARGEFUNNEL              100
 
+/**
+ * Flags used in te_create_large_funnel()
```

Comment:
Why are these not in an enumeration as well?

Reply:
@rsKliPPy Not sure to be honnest. I changed them now.